container/container.go

```diff
@@ -270,6 +271,30 @@ func NetworkForTask(taskInfo *mesos.TaskInfo) string {
 	return networkMode
 }
 
+func LogOpsForTask(taskInfo *mesos.TaskInfo) docker.LogConfig {
+
+	var logDriver string
+	for _, param := range getParams("log-driver", taskInfo) {
+		logDriver = *param.Value
+	}
+
+	logOps := make(map[string]string)
+	for _, param := range getParams("log-opt", taskInfo) {
+		values := strings.SplitN(*param.Value, "=", 2)
+		if len(values) < 2 {
+			log.Debugf("Got label with empty value: %s", *param.Key)
+			continue // No empty labels
```

Comment:
the log statement above says that the label has an empty value, but the comment contradicts it. Not sure I get it...